Source/API/population.h

```diff
@@ -9,13 +9,13 @@ class population {
 public:
     virtual ~population() = default;
 
-    virtual size_t get_size() = 0;
+    virtual size_t get_size() const = 0;
 
-    virtual creature* operator[] (int) = 0;
+    virtual creature* operator[] (int) const = 0;
```

Comment:
I think the return type should be `const creature* const` or at the very least `creature* const`.

Reply:
Your right some type of const modifier would make sense here. The thing is the pointer audit will likely change the return type so I'm not sure yet what the modifier would look like. I was thinking maybe we leave this half baked like this and take care of this in the pointer branch. What do you think of that?